website/docs/getting-started/setup/instance-configuration/email/README.md

```diff
@@ -7,76 +7,88 @@ sidebar_position: 2
 
 # Email
 
-[Email](https://en.wikipedia.org/wiki/Email) is a widely used service to communicate with your users. You can set up email integration on your Appsmith instance to:
+This page guides you through configuring an email service provider on your self-hosted instance. By configuring email, you can leverage the below capabilities:
 
-* Invite users to your Appsmith workspace
-* Notify admins of important events & approval requests
-* Handle user management emails such as invites and password resets
+* Invite users to join your Appsmith workspace by sending email invitations.
+* Reset password using password reset emails.
 
-:::note
-[Password resets](/help-and-support/troubleshooting-guide/application-errors#reset-password-error) for user accounts are handled by emails from the Appsmith instance. If you would like users to be able to perform this action, ensure that your instance has been configured to handle email notifications. 
-:::
+<VideoEmbed host="youtube" videoId="NOAofPbmJWw" title="Configure Email Service Provider." caption="Configure Email Service Provider" /> 
 
-Appsmith allows you to configure email using environment variables or the [admin settings](/getting-started/setup/instance-configuration).
+## Prerequisites
```

Comment:
this section doesn't give any information that is not already known. From the introduction, I think it's known that they need an Appsmith instance and an email service account. Do we need to mention these explicitly?  We don't mention this as prerequisites for any instance configuration guides.

Reply:
We do use it. We've used it in the:
* [External MongoDB](https://docs.appsmith.com/getting-started/setup/instance-configuration/custom-mongodb-redis#prerequisites).
* [Google Maps](https://docs.appsmith.com/getting-started/setup/instance-configuration/google-maps#prerequisites)

Reply:
They need to be able to access the instance to configure.
May be I can rewrite it as superuser.

Reply:
This is added to the steps so removed the prerequisite section now.